net/lp_rpc.pb.go

```diff
@@ -22,7 +43,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
```

Comment:
Hm looks like a lower proto package version was used to generate this file - can we maintain the same package version by using the version 3 for the locally installed package?